modules/local/kraken2.nf

```diff
@@ -28,7 +27,7 @@ process KRAKEN2 {
         def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
         """
         kraken2 \\
-            --db $db \\
+            --db ${params.kraken2_db} \\
```

Comment:
Will this be mounted properly inside a docker container if not specified as a path to the process?

Reply:
fixed, I forgot about that